project-management/src/main/java/life/qbic/projectmanagement/application/authorization/QbicUserDetails.java

```diff
@@ -57,6 +57,10 @@ public String getUsername() {
     return user.id();
   }
 
+  public String getFullName() {
```

Comment:
NIT: 
```suggestion
  public String fullName() {
```
for readability